contrib/lib/src/space_helmet/mod.rs

```diff
@@ -0,0 +1,82 @@
+//! Space Helmet is a [Fairing](../../rocket/fairing/trait.Fairing.html) that
+//! turns on browsers security features via. HTTP headers.
+//!
+//! It provides a typed interface to help prevent configuration errors at
+//! compile time and resonable defaults for many applications.  It takes some
+//! inspiration from [helmet](https://helmetjs.github.io/), a similar piece
+//! of middleware for [express](https://expressjs.com).
+//!
+//! #### What does it support?
+//!
+//!|  | HTTP Header| Enabled by Default?|
+//! --- | --- | ---
+//!| [xss_protect](struct.SpaceHelmet.html#method.xss_protect) - prevents some xss attacks.| [X-XSS-Protection][xss_url]           |  ✔ |
```

Comment:
This isn't particularly legible without rendering the table. Here's what would work better:

```markdown
| Method                             | Description                            | HTTP Header                 | Enabled by Default? |
| ---------------------------------- | -------------------------------------- | --------------------------- | ------------------- |
| [`SpaceHelmet::xss_protect()`]     | Prevents some reflected XSS attacks.   | [X-XSS-Protection]          | ✔                   |
| [`SpaceHelmet::no_sniff()`]        | Prevents client sniffing of MIME type. | [X-Content-Type-Options]    | ✔                   |
| [`SpaceHelmet::frameguard()`]      | Prevents clickjacking.                 | [X-Frame-Options]           | ✔                   |
| [`SpaceHelmet::hsts()`]            | Enforces strict use of HTTPS.          | [Strict-Transport-Security] | ?                   |
| [`SpaceHelmet::expect_ct()`]       | Enables certificate transparency.      | [Expect-CT]                 | ✗                   |
| [`SpaceHelmet::referrer_policy()`] | Enables referrer policy.               | [Referrer-Policy]           | ✗                   |

[X-XSS-Protection]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection
[X-Content-Type-Options]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options
[X-Frame-Options]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options
[Strict-Transport-Security]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
[Expect-CT]:  https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT
[Referrer-Policy]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy
```